dynawo/sources/Models/Modelica/Dynawo/Examples/HVDC/HVDC.mo

```diff
@@ -18,35 +18,32 @@ model HVDC "HVDC link connected to two infinite buses"
 
   extends Icons.Example;
```

Comment:
You have changed a lot of things in this test case, are all these changes necessary ? Does the case work properly ?